tests/h264Testing/h264Verifier.js

```diff
@@ -0,0 +1,95 @@
+const Promise = require('bluebird');
+const exec = require('child_process').exec;
+const fs = require('fs');
+require('../../lib/utils/KalturaConfig');
+require('../../lib/utils/KalturaCache');
+
+class h264Verifier
+{
+	/**
+	 * @param flavorIds - array of flavor ids
+	 * @param flavorPaths - array of flavor paths
+	 */
+	static insertFlavorsPathToCache(flavorIds, flavorPaths)
+	{
+		h264Verifier.writeToLog(`called insertFlavorsPathToCache with ${flavorIds} and ${flavorPaths}`, 'h264.log');
+		const testTime = KalturaConfig.config.h264Verfication.cacheTime;
+		for (let i = 0; i < flavorIds.length; i++)
+		{
+			KalturaCache.set(flavorIds[i], { path: flavorPaths[i] }, testTime,
+				function () {
+					h264Verifier.writeToLog(`inserted flavorId ${flavorIds[i]} with path ${flavorPaths[i]} to cache`, 'h264.log');
+				},
+				function () {
+					h264Verifier.writeToLog(`unable to insert flavorId ${flavorIds[i]} with path ${flavorPaths[i]} to cache`, 'h264.log');
+				}, null);
+		}
+	}
+
+	static verifyAd(flavorId, adPath)
+	{
+		h264Verifier.writeToLog(`called verifyAd with ${flavorId} and ${adPath}`, 'h264.log');
+		KalturaCache.get(flavorId,
+			function (flavorPath) {
+				h264Verifier.writeToLog(`got flavorId ${flavorId} from cache`, 'h264.log');
+				h264Verifier.runVerification(flavorId, flavorPath.path, adPath);
+			},
+			function (err) {
+				h264Verifier.writeToLog(`faild to get flavorId ${flavorId} from cache`, 'h264.log');
+				h264Verifier.writeToLog(`got err - ${err}`, 'h264.log');
+			});
+	}
+
+	/**
+	 * runs verification on flavor and ad

```

Comment:
Add  the following to this function description 
Using script A to verify that X is like Y 
and using script A to that Z is like M